.github/workflows/snap.yaml

```diff
@@ -24,6 +24,8 @@ jobs:
       snap-names: ${{ steps.snap-names.outputs.snap-names }}
     steps:
       - uses: actions/checkout@v3
+        with:
+          ref: '${{ inputs.branch-name }}'
```

Comment:
When `branch-name == ''`, as by the default case, that results in checking out the default branch right?

Reply:
not it was not, I changed that.